src/main.rs

```diff
@@ -303,6 +325,26 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
                 Err(_) => "".to_owned(),
             }
         }
+        Command::PostComment { message, secret } => {
+            let client = reqwest::blocking::Client::new();
+            let github_event_path = env::var("GITHUB_EVENT_PATH")?;
+            let v: Value = serde_json::from_str(&github_event_path)?;
+            let uri = v["pull_request"]["comments_url"].as_str();
```

Comment:
It's possible that "pull_request" doesn't exist if the command was executed in another type of event. In that case it would be nice if the error would be explicit about: `did not find the comments_url, this command only works inside of "pull_requests" events.`